r/R/arrow-package.R

```diff
@@ -153,6 +153,9 @@ arrow_with_dataset <- function() {
 #' @rdname arrow_available
 #' @export
 arrow_with_parquet <- function() {
+  # https://github.com/google/snappy/pull/148/files
+  skip_on_linux_devel()
```

Comment:
This doesn't work: that function is in a test helper, and it requires testthat

Reply:
Also I don't think we have to gate all of parquet support on this, only snappy

Reply:
Right, right. I'll pull it out into a separate helper (the renaming of the skip makes it more accurate with what it does though so I'm inclined to keep that)

And, yeah we don't need to gate all of parquet, though anywhere we don't set a default will use snappy. I'm not 100% sure why [only the example](https://dev.azure.com/ursacomputing/crossbow/_build/results?buildId=16490&view=logs&j=0da5d1d9-276d-5173-c4c4-9d4d4ed14fdb&t=d9b15392-e4ce-5e4c-0c8c-b69645229181&l=12300) runs into the UBSan, possibly the shape of data / how we are projecting is unique there?